src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -599,8 +595,13 @@
     <Field Name="Checked" Type="bool"/>
     <Field Name="ExplicitCastInCode" Type="bool"/>
     <Field Name="ConstantValueOpt" Type="ConstantValue" Null="allow"/>
-    <!-- True for explicit casts with top-level `?`. Used by NullableWalker. -->
-    <Field Name="IsExplicitlyNullable" Type="bool" Null="allow"/>
```

Comment:
>IsExplicitlyNullable [](start = 17, length = 20)

I am surprised you don't need this.

Reply:
This field has moved to `ConversionGroup`, as `ExplicitType`.

---
In reply to: [190738762](https://github.com/dotnet/roslyn/pull/26356#discussion_r190738762) [](ancestors = 190738762)